docs/install.rst

```diff
@@ -0,0 +1,43 @@
+Installing ``uvcombine``
+============================
+
+Requirements
+------------
+
+This package has the following dependencies:
+
+* `Python <http://www.python.org>`_ 3.6 or later (Python 3.x is supported)
+* `Numpy <http://www.numpy.org>`_ 1.8 or later
+* `Astropy <http://www.astropy.org>`__ 1.0 or later
+* `six <http://pypi.python.org/pypi/six/>`__
+
+Installation
+------------
+
+..
+    To install the latest stable release, you can type::
+
+..
+    pip install uvcombine
```

Comment:
do we not have a pypi version going?

Reply:
Not yet.

Reply:
Should wait until we finish the tests, IMO.